src/Classes/Loader.ts

```diff
@@ -140,6 +175,31 @@ export class Loader {
         }
     };
 
+    /**
+     * Loads the provided json with a GET request. The content will be used to create a [[Container]] object.
+     * This method is async, so if you need a callback refer to JavaScript function `fetch`
+     * @param id the particles container element id
+     * @param domContainer the container used to contains the particles
+     * @param jsonUrl the json path to use in the GET request
+     */
+    public static async setJSON(id: string, domContainer: HTMLElement, jsonUrl: string): Promise<Container | undefined> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)